speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/CHANGES.txt

```diff
@@ -58,16 +60,27 @@ Changed:
                 added backend
             grsound.cpp
                 added intervention sound initialization
+                added intervention sound volume setting
             OpenalSound.cpp
                 added check for playing state
+            OpenalSoundInterface.cpp
+                added UpdateInterventionSounds()
             OpenalSoundInterface.cpp
                 added UpdateInterventionSounds()
                 added update and starting of intervention sounds
             SoundInterface.cpp
                 added setInterventionSound()
+                added SetInterventionVolume()
+                added GetInterventionVolume()
+                added InterventionVolume member to init
```

Comment:
If its a change in a function, start with changed instead of added
```suggestion
                changed SoundInterface(): added InterventionVolume member init
```